CIFAR10/train/fastai_pytorch_single_volta.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": "v1.0",
+    "author": "fast.ai + students team: Jeremy Howard, Andrew Shaw, Brett Koonce, Sylvain Gugger",
+    "authorEmail": "j@fast.ai",
+    "framework": "fastai / pytorch",
+    "codeURL": "https://github.com/fastai/imagenet-fast/commit/a2ff907cd5e7315088324e1ff324fd25a1a67fe3",
+    "model": "Custom Wide Resnet",
+    "hardware": "Paperspace Volta (V100)",
+    "costPerHour": 2.30,
+    "timestamp": "2018-04-19",
+    "misc": {
+        "model implementation": "https://github.com/fastai/imagenet-fast/blob/e93593ca0a7e226141050cbd55622ff08b0632c3/cifar10/models/wideresnet.py#L81",
```

Comment:
The commit hash in this link doesn't match the hash in the `codeURL` above. Were there any significant changes to the code or scripts you ran? I glanced over the commit history of your repo, and it looks like the changes between these two commits were for documentation. I just want to confirm.

Reply:
see note in #23

Reply:
Thanks!